docs/claws-mail.md

```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
```

Comment:
The relative links will have to be fixed when I convert to rst. I am not sure if there's appropriate syntax in Markdown, but it'll need fixing anyway.